### PR TITLE
MFT: application of physics cuts and flags

### PR DESCRIPTION
**Pull request by bovulpes**
MFT needs to be able to modify physics cuts and processes flags for the materials built by the MFT as well as for three materials built in the part of the code common with ITS, indicated as ALPIDE. This needs a small change in the MaterialManager. The MFT simcuts.dat file is modified accordingly. The final values are under study.

Some bugs are fixed in the MFT creation of materials and some simplifications are done in the local indices of the materials and media. 

See also the discussion at https://alice.its.cern.ch/jira/browse/O2-696

### CODE REVIEW

File: Detectors/Base/src/MaterialManager.cxx [hunk 1]
```diff
@@ -405,14 +405,14 @@ void MaterialManager::loadCutsAndProcessesFromFile(const char* modname, const ch
     // apply cuts via material manager interface
     for (int i = 0; i < NCUTS; ++i) {
       if (cut[i] >= 0.) {
-        SpecialCut(modname, itmed, cutnames[i], cut[i]);
```
Comment (sawenzel) [inline Detectors/Base/src/MaterialManager.cxx:408]: Detector name and module name are probably the same thing. Only that one is passed to the function and the other is read from the cut file.
In your case, this now allows to specify cuts for ALPIDE from within MFT which is an improvement.

I am still wondering about the following:
The cuts you set for ALIPE ... will they have an impact on other users of ALPIDE such as ITS? Or can ITS define their own ALIPDE cuts?

If not ... probably we still need to go further and see ALPIDE as a template that can be customized in concrete instantiations..

I'll merge this for now but's let's keep an eye on it.

**Comment by sawenzel** [comment]
Mm ... some sim tests are failing.

**Comment by sawenzel** [comment]
There is a geometry crash: @bovulpes : Is this volume part of MFT?

-------- EEEE ------- G4Exception-START -------- EEEE -------
*** G4Exception : G4Root_F003
      issued by : TG4RootDetectorConstruction::CreateG4LogicalVolume
      Cannot make material for volume: pipeTube10_D0_H0

*** Fatal Exception *** core dump ***
 **** Track information is not available at this moment
 **** Step information is not available at this moment

-------- EEEE -------- G4Exception-END --------- EEEE -------

File: Detectors/Base/src/MaterialManager.cxx (hunk 1 above)
Comment (bovulpes) [reply Detectors/Base/src/MaterialManager.cxx:408]: That's right, if MFT creates the materials and defines the cuts for ALPIDE, ITS will retrieve the materials and overwrite the cuts and flags. In principle, MFT and ITS should agree on the best values to use.

**Pull request merged**